### PR TITLE
Add issue management to pom file

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -50,6 +50,11 @@
             <url>https://repo.jenkins-ci.org/releases</url>
         </repository>
     </distributionManagement>
+    
+    <issueManagement>
+        <system>GitHub</system>
```
Comment (timja) [inline pom.xml]:
```suggestion
        <system>GitHub Issues</system>
```

**Comment by timja** [comment]
May not merge this, the GitHub api exposes whether issues are enabled, probably better to not bother adding the issue tracker here

**Comment by casz** [comment]
True that 😅

**Comment by timja** [comment]
Closing in favour of https://github.com/jenkins-infra/update-center2/pull/331 which doesn't require pom changes

**Pull request closed without merging**